cypress/integration/homePageView_spec.js

```diff
@@ -0,0 +1,51 @@
+
+describe('User visits the homepage tests', () => {
+
+    beforeEach(() => {
+        cy.visit('http://localhost:3000');
+      });
+
+    it('Should show the user a page title and directive text', () => {
+      cy.contains('h1', 'Rancid Tomatillos')
+    });
+
+    it('Should show the user a selection of movies from a database upon page load', () => {
+        cy.fixture('movies.json')
```

Comment:
I wonder if we can save this fixture in the beforeEach to run before every test!